signup/signup.js

```diff
@@ -59,30 +63,72 @@ function pwcheckToggleIcon(event) {
   input.togglePasswordType(signupPwCheckInput, signupPwcheckIcon);
 }
 
-//엔터키
-function handlesignupButtonEnter(event) {
-  if (event.code === 'Enter') {
-    handlesignupButtonClick();
-  }
-}
+//버튼 part
+const signupButton = document.querySelector('.signup__form--signupBt');
+signupButton.addEventListener('click', handlesignupButtonClick);
 
 //로그인 버튼 클릭 시
-function handlesignupButtonClick(event) {
-  if(signupEmailInput.value !== '' && signupEmailInput.value !== '' && signupEmailInput.value !== '' ){
-    if (signupEmailInput.value !== 'test@codeit.com') {
-      if (signupPwInput.value === signupPwCheckInput.value) {
+async function handlesignupButtonClick(event) {
+  //input이 비어있을 때
+  if (
+    signupEmailInput.value === "" ||
+    signupPwInput.value === "" ||
+    signupPwCheckInput.value === ""
```

Comment:
조건에 많을때는 이름을 붙여주는게 좋습니다. 
```suggestion
    const isInvalidInput =      signupEmailInput.value === "" ||
    signupPwInput.value === "" ||
    signupPwCheckInput.value === ""

      if(isInvalidInput)
```